core-web/apps/dotcms-ui/src/app/shared/dot-custom-reuse-strategy/dot-custom-reuse-strategy.service.ts

```diff
@@ -14,11 +14,12 @@ export class DotCustomReuseStrategyService implements RouteReuseStrategy {
     }
 
     shouldReuseRoute(future: ActivatedRouteSnapshot, curr: ActivatedRouteSnapshot): boolean {
-        return future.routeConfig === curr.routeConfig
-            ? future.data.reuseRoute === false
-                ? false
-                : true
-            : false;
+        if (future.routeConfig !== curr.routeConfig) {
+            return false;
+        }
```

Comment:
Not sure what kind of data is `routeConfig` but if they are objects, this will always enter, since they are different references

Reply:
I didn't changed this; I simply made it more readable. The conditions remain the same.

Reply:
From what I read, it is usually `null`. For some specific routes the `routeConfig` is an actual objects and for them, we need to reload